### PR TITLE
correct use of staging versus production Molecule scenarios

### PR DESCRIPTION
**Pull request by cfm**
Towards #3:

1. Default to setting up a proper SecureDrop [staging environment][staging] via `make build-debs && make staging`.
2. Offer "expert" instructions for using this setup as the basis for a production VM environment, [installed](https://docs.securedrop.org/en/stable/development/virtual_environments.html#install-from-an-admin-workstation-vm) from the preconfigured Tails domain.

## Testing

1. [ ] `terraform destroy` if you already have an `sd-staging` box running.
2. [ ] `terraform apply`
3. [ ] Wait ~20 minutes for cloud-init to complete `make build-debs && make staging`.
4. [ ] Confirm that `app-staging` is fully configured by checking for the [expected onion services][staging]:
    > The web interfaces and SSH are available over Tor. A copy of the the Onion URLs for *Source* and *Journalist Interfaces*, as well as SSH access, are written to the Vagrant host’s `install_files/ansible-base` directory.

[staging]: https://docs.securedrop.org/en/stable/development/virtual_environments.html#staging

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,49 @@ If you used the SSH invocation above, your Tails domain will be available via
...
+
+To do so, pick up the instructions from the section ["Install from an
+Admin Workstation VM"][install-from-admin-workstation-vm].  First,
+provision the production VMs alongside the existing staging VMs:
+
+```sh-session
+$ ssh -L 5900:localhost:5902 root@<your IP>
+[...]
+root@sd-staging:~# cd securedrop
+root@sd-staging:~/securedrop# molecule create -s libvirt-prod-focal
```
Comment (eloquence) [inline README.md:87]: `molecule` was not available to me until I activated the virtualenv via `source .venv/bin/activate`.

**Comment by eloquence** [comment]
Nit above aside (and using the new Tails image pulled in via #6), this worked perfectly for me. Here's the Tails VM accessing SI and JI set up via prod VMs:

Screenshot from 2021-12-16 00-26-30

File: README.md (hunk 1 above)
Comment (cfm) [reply README.md:88]: Thanks!  Added in 87361d6.

**Comment by cfm** [comment]
Thanks for testing, @eloquence!

**Pull request merged**